lunatrace/bsl/backend-cdk/package.json

```diff
@@ -45,5 +45,8 @@
     "ts-jest": "^26.2.0",
     "ts-node": "^10.4.0",
     "typescript": "~3.9.7"
+  },
+  "dependencies": {
+    "aws-cdk-datadog-ecs-integration": "~0.2.2"
```

Comment:
looks like all other deps here are devdeps, should we move this for consistency?